<launch>
	<!-- Config file with initial velocities for each robot. -->
	<arg name="filename" default="$(find boids_ros)/cfg/sphero_init_vel.cfg"/>

	<!-- Start Reynolds controller nodes launcher. -->
	<node pkg="boids_ros" type="reynolds_launch.sh" name="reynolds_launcher" args="$(arg filename) robot" output="screen"/>

	<!-- Start nearest_search node: search for other robots in range. -->
	<node pkg="boids_ros" type="nearest_search.py" name="search" output="screen">
		<param name="robot_name" type="string" value="robot"/>
	</node>

	<!-- Record a bag for debug purposes -->
	<!-- <arg name="rosbag_args" default='-O $(find boids_ros)/bagfiles/sim_test.bag -e "(.*)/odom" '/> -->
	<!-- <arg name="rosbag_args" default='-O $(find boids_ros)/bagfiles/kalman_test.bag /robot_0/odom /robot_0/debug_est'/> -->
	<!-- <node pkg="rosbag" type="record" name="rosbag_record" args="$(arg rosbag_args)" output="screen"/> -->
</launch>
